x-pack/plugins/apm/common/alert_types.ts

```diff
@@ -7,12 +7,30 @@
 
 import { i18n } from '@kbn/i18n';
 import type { ValuesType } from 'utility-types';
+import * as moment from 'moment';
+import momentDurationFormatSetup from 'moment-duration-format';
 import type { AsDuration, AsPercent } from '../../observability/common';
 import type { ActionGroup } from '../../alerting/common';
 import { ANOMALY_SEVERITY, ANOMALY_THRESHOLD } from './ml_constants';
 
+momentDurationFormatSetup(moment);
 export const APM_SERVER_FEATURE_ID = 'apm';
 
+const getInterval = (windowSize: number, windowUnit: string): string => {
```

Comment:
@fkanout Looks good. We could possibly move this later on to the `observability` plugin, because we need the same functionality for rest reason messages. I do something similar with uptime

Reply:
@mgiota, sure! Maybe you mean copping it, but as far I'm concerned, we can't share code across plugins/Apps

Reply:
Anything exported in the obs plugin can be imported by APM (but not vice versa, I think)

Reply:
@dgieselaar thanks for the info! I will check that and file an issue for it.

Reply:
@fkanout Yep as discussed at the moment I have same code in uptime plugin. Once you move this sharable code under Observability plugin then I will import it in uptime from obs plugin.